### PR TITLE
Script for sending out a stats update email

### PR DESCRIPTION
**Pull request by Athight**
includes attempted forced newlines.
~needs cronjob on the server to be set once this is deployed.~
Decided to manually deploy the one file and set up the cronjob now.

### CODE REVIEW

File: src/statsupdate.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	part['Content-Disposition'] = 'attachment; filename="' + filename + '"'
+	smtp_msg.attach(part)
+
+	m._send_msg(smtp_msg)
+
+def makeData(data):
+	compiledData = "date match generated, USD value, charity1, country1, charity2, country2\n"
+
+	for match in data:
+		compiledData += "{}, {}, {}, {}, {}, {}\n".format(match['created_ts'], match['value'], match['details']['new_offer_charity'], match['details']['new_offer_country'], match['details']['old_offer_charity'], match['details']['old_offer_country'])
```
Comment (bigfighter) [inline src/statsupdate.py]: one of these might be more readable
 * `'%(created_ts)s, %(value)s, ...' % match` (doesn't work so well for sub-sub thingies
 * `f"{match['created_ts']} {match['value']}..."` (depends on installed python versions)

Comment (Athight) [reply src/statsupdate.py]: I believe the % operator for strings isn't intended to continue into the future.
Happy to try and rewrite it cleaner if you'd prefer, but i figured I'd go with something basic that shouldn't complain if something changes

Comment (Athight) [reply src/statsupdate.py]: (merging, feel free to ping me any time to fix this)

**Comment by Athight** [comment]
merging into master

**Comment by Athight** [comment]
err reopen?

**Pull request merged**